core/include/vecmem/utils/impl/copy.ipp

```diff
@@ -395,6 +398,138 @@ copy::event_type copy::set_sizes(
     return create_event();
 }
 
+template <typename SCHEMA>
+copy::event_type copy::setup(edm::view<SCHEMA> data) const {
```

Comment:
NIT: `init()`?

Reply:
Hmm... I do actually like `init(...)` better than `setup(...)`. But we already have 2 `vecmem::copy::setup(...)` functions for 1D and jagged vectors. :thinking:

In this PR I would stay with the `setup(...)` name. But I'm absolutely open to renaming it to `init(...)` at one point. At that point we'll have to update all the client code. (`setup(...)` is called in a lot of places in [traccc](https://github.com/acts-project/traccc).)